ideals.html

```diff
@@ -151,6 +151,12 @@
 	<p>Cockpit allows others to add additional <span class="caps">UI</span> pieces. We do
 	not allow arbitrary extension of the <span class="caps">UI</span> all over the place, but
 	at distinct extension&nbsp;points.</p>
+
+	<h2>Cockpit is&nbsp;tested</h2>
+	<p>Every feature and change in Cockpit is covered by an automatic test.
```

Comment:
We certainly are far from 100% coverage, but in the spirit of ["Ideals are like stars..."](https://www.brainyquote.com/quotes/quotes/c/carlschurz389236.html) we generally try to abide by it and reject new features/fixes without tests.